scripts/check_configs.ts

```diff
@@ -426,6 +443,35 @@ function isValidMarketPromo(marketPromo: {[marketId: string]: MarketPromo}, netw
   return true;
 }
 
+function isValidTransferBanner(transferBanner: TransferBanner, network: CarbonSDK.Network): boolean {
+  const { temporary_disabled_transfer_tokens, temporary_disabled_bridges } = transferBanner;
+  if (Object.keys(temporary_disabled_transfer_tokens).length > 0) {
+    Object.keys(temporary_disabled_transfer_tokens).map((key) => {
+      const { start, end } = temporary_disabled_transfer_tokens[key];
+      const startTime = new Date(start);
+      const endTime = new Date(end);
+      if (endTime < startTime) {
+        console.error(`ERROR: ${network}.json has invalid end time (${end}) is before start time (${start}) for token denom ${key}.`);
+        return false;
+      }
+    });
+  }
+
+  if (Object.keys(temporary_disabled_bridges).length > 0) {
+    Object.keys(temporary_disabled_bridges).map((key) => {
```

Comment:
1. I see `Object.keys(temporary_disabled_bridges)` is used twice, pls store this in a variable and apply where necessary.
i.e.
```typescript
const disabledBridgesKeys = Object.keys(temporary_disabled_bridges)
if (disabledBridgesKeys.length > 0) {
  disabledBridgesKeys.forEach((key) => { ... })
}
```

2. Pls check that each bridge key is valid. Please refer to this [code snippet](https://github.com/Switcheo/demex-webapp-config/blob/master/scripts/check_configs.ts#L288) to see how to check if the bridge names are valid.